tests/scripts/thread-cert/Cert_5_5_05_SplitMergeREED.py

```diff
@@ -101,7 +101,7 @@ def test(self):
         self.nodes[ROUTER3].stop()
         time.sleep(140)
 
-        self.assertEqual(self.nodes[ROUTER1].get_state(), 'child')
+        self.assertEqual(self.nodes[ROUTER1].get_state(), 'router')
```

Comment:
I think routers should only try to maintain their status if a router has children attached.  In this test case, I believe Router_1 should not have any children attached.  If no children are attached, then the device should adhere to the usual router upgrade checks before becoming a router.  Thoughts?